config/config.go

```diff
@@ -857,6 +857,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.gumgum.endpoint", "https://g2.gumgum.com/providers/prbds2s/bid")
 	v.SetDefault("adapters.improvedigital.endpoint", "http://ad.360yield.com/pbs")
 	v.SetDefault("adapters.inmobi.endpoint", "https://api.w.inmobi.com/showad/openrtb/bidder/prebid")
+	v.SetDefault("adapters.interactiveOffers.endpoint", "https://rtb.ioadx.com/bidRequest/?partnerId=d9e56d418c4825d466ee96c7a31bf1da6b62fa04")
```

Comment:
I'd like to verify the partner id refers to the Prebid ecosystem and not a specific host, such as Xandr/AppNexus.

Reply:
Correct, that partner id is regarding the Prebid ecosystem